src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTable.cs

```diff
@@ -20,16 +20,25 @@ public FileRoutingTable(string filePath, TimeSpan checkInterval, IAsyncTimer tim
             this.timer = timer;
             this.fileAccess = fileAccess;
             this.maxLoadAttempts = maxLoadAttempts;
+
+            errorMessage = $"Error while reading routing table file at {filePath}.";
```

Comment:
Maybe something like "An error occured while reading the endpoint instance mapping file at {filePath}. Please check the routing documentation and see the inner exception for more details." ?